pyproject.toml

```diff
@@ -12,3 +12,26 @@ requires = [
 # Defined by PEP 517
 build-backend = "setuptools.build_meta"
 
+[tool.repo-review]
+# These are a list of the currently failing tests:
+ignore = [
+  "PY005",  # Has tests folder --
+  "PP003",  # Does not list wheel as a build-dep --
+  "PP301",  # Has pytest in project --
+  "PP304",  # Sets the log level in pytest s
+  "PP305",  # Specifies xfail_strict s
+  "PP306",  # Specifies strict config s
+  "PP307",  # Specifies strict markers s
+  "PP309",  # Filter warnings specified s
+  "GH212",  # Require GHA update grouping --
+  "PC110",  # Uses black or ruff-format --
+  "PC140",  # Uses a type checker --
+  "PC160",  # Uses a spell checker --
+  "PC170",  # Uses PyGrep hooks (only needed if rST present) --
+  "PC180",  # Uses a markdown formatter --
+  "PC190",  # Uses Ruff --
+  "PC901",  # Custom pre-commit CI message --
+  "MY100",  # Uses MyPy (pyproject config) --
+  "RF001",  # Has Ruff config --
+]
```

Comment:
Please can each of these include a URL to the documentation for this error, which is what we have done with other repos e.g.

- [Geovista](https://github.com/bjlittle/geovista/blob/36ea0db7005218897cf38f4a742c5757196e9314/pyproject.toml#L143-L151)
- [Iris-grib](https://github.com/SciTools/iris-grib/blob/d00965d4b411e12c4e6a166ee7903a2d1129c05e/pyproject.toml#L137-L149)

(I think I remember specific discussions about this, although I'm a bit hazy. That would suggest it's something we care about).